docs/guides/vtex-io/Storefront-Guides/getting-started-with-storefront-solutions.md

```diff
@@ -0,0 +1,46 @@
+| **Tech stack** | * React<br>* TypeScript<br>* Node.js<br>* Next.js<br>* GraphQL<br>* VTEX WebOps | * React<br>* TypeScript<br>* Node.js<br>* .NET<br>* GraphQL<br>* VTEX IO | * CSS<br>* HTML<br>* JavaScript |
+| **Starter (initial template)** | After completing the [FastStore Starter](https://starter.vtex.app/), you can access the [FastStore Onboarding](https://www.faststore.dev/docs/getting-started-overview). This allows you to kick off the project with a base template focused on performance. | The [Store Theme](https://github.com/vtex-apps/store-theme) is the initial template you can use to launch the storefront project or start from scratch. | The [Layout](https://help.vtex.com/es/tracks/cms--2YcpgIljVaLVQYMzxQbc3z/1oN446gRGcR2s70RvBCAmj) section allows you to create your storefront with HTML and CSS.<br><br>Inside it, you can find the standard [templates](https://help.vtex.com/en/tutorial/what-are-templates--4l7BQBYO9ycumsqua2CU88), which are files that contain the code of your website pages. |
+| **CMS solution** | Integrates with the [Headless CMS](https://developers.vtex.com/docs/guides/faststore/headless-cms-overview). | Integrates with the [Site Editor](https://developers.vtex.com/docs/guides/vtex-io-documentation-site-editor). | The [Layout](https://help.vtex.com/en/tutorial/what-is-cms-layout--EmO8u2WBj2W4MUQCS8262) feature allows you to create your storefront with HTML and CSS. |
+| **Apps Admin (Extension Hub)** | There is no integration with the VTEX app platform. | Allows installation and use. | Allows installation and use of some apps. |
+| **Customizations** | Performance-focused: Easier to add customizations without degrading performance. | Composability-focused: Flexible and customizable through VTEX IO apps. | Customizations are limited and can be managed by creating, organizing, and editing the structure of folders, files, and components within the Layout feature. |
+| **Performance** | Maximum performance possible, as it uses [Jamstack](https://jamstack.org/), which serves pre-rendered content to a CDN. It is made dynamic through APIs and serverless functions. | SSR (Server Side Rendering) features generate the complete HTML of a page on the server as a response to a page request, while SPA (Single Page Application) loads elements such as the header only once the page is loaded. | Outdated technologies make it challenging to maintain optimal performance, especially in scenarios demanding multiple customizations. |
+| **Availability** | Installation is available upon request through [VTEX Support](https://help.vtex.com/en/support). For information on unavailable features, refer to the \[Features and Functionalities\](#features-and-funcionalities) topic. | General availability. | No longer available to new stores starting with VTEX. |
```

Comment:
Como no [VTEX Vision](https://vision.vtex.com/spring-24/) foi anunciado como `General availability for LATAM, Brazil, and  North America; limited access for other regions`, podemos adicionar essa parte. Algo como:

> Generally available for LATAM, Brazil, and  North America; limited access for other regions. For information on unavailable features, refer to the \[Features and Functionalities\](#features-and-funcionalities) topic.